pkg/console/subresource/deployment/deployment.go

```diff
@@ -229,24 +229,20 @@ func consoleVolumeMounts(vc []volumeConfig) []corev1.VolumeMount {
 	return volMountList
 }
 
+type LogMapping map[operatorv1.LogLevel]string
```

Comment:
I think I realized my suggestion here may also be incorrect, the more indiomatic way to do this is prob like:
https://github.com/openshift/console-operator/pull/212/files#r278728198

Reply:
a `case` can match several items and is substantially faster.